ULiege-Droit.csl

```diff
@@ -0,0 +1,520 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="fr-FR" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>ULiège - Droit (French)</title>
+    <title-short>ULiège-dt</title-short>
+    <id>http://www.zotero.org/styles/ULiege-Droit</id>
+    <link href="http://www.zotero.org/styles/universite-libre-de-bruxelles-histoire" rel="template"/>
```

Comment:
@graulich, thanks for the submission. Before you do any more work, is there a style guide available for this citation format, either online or in print, that we could link to?